src/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+moment().format();
```

Comment:
Not sure what this line is doing at the moment (line 3) - did you leave this in by accident ?